commons-db/src/main/java/com/palantir/logger/SqlStackLogWrapper.java

```diff
@@ -1,50 +0,0 @@
-/**
- * Copyright 2015 Palantir Technologies
- *
- * Licensed under the BSD-3 License (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://opensource.org/licenses/BSD-3-Clause
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.palantir.logger;
-
-public class SqlStackLogWrapper {
```

Comment:
I also found this class being used in internal product as a note, happy to keep in if you are also pruning it from there.